configure.ac

```diff
@@ -1853,7 +1853,7 @@ AC_DEFINE_UNQUOTED(ENABLE_SYSTEMD, `enable_value $enable_systemd`, [Enable syste
 AC_DEFINE_UNQUOTED(ENABLE_KAFKA, `enable_value $enable_kafka`, [Enable kafka support])
 AC_DEFINE_UNQUOTED(SYSTEMD_JOURNAL_MODE, `journald_mode`, [Systemd-journal support mode])
 AC_DEFINE_UNQUOTED(HAVE_INOTIFY, `enable_value $ac_cv_func_inotify_init`, [Have inotify])
-AC_DEFINE_UNQUOTED(HAVE_GETRANDOM, `enable_value $ac_cv_func_getrandom`, [Have getrandom])
+#AC_DEFINE_UNQUOTED(HAVE_GETRANDOM, `enable_value $ac_cv_func_getrandom`, [Have getrandom])
```

Comment:
Please remove the whole line if it is not needed. Also could you kindly elaborate more, why do we have to remove this?

Reply:
It is used by the `threaded-random-generator` example plugin.

Reply:
This instance of `AC_DEFINE_UNQUOTED` must be removed: getrandom is detected using AC_CHECK_FUNCS too. This line otherwise messes up the already set compiler switch. I will move its removal into a separate commit.

Reply:
I could not find a good way to separate it, so I just squashed and added a note in the commit message.

Reply:
Thanks! 👍